Formula/csound.rb

```diff
@@ -37,6 +38,21 @@ class Csound < Formula
     sha256 "2716e916a9dd9445b2a4de1f2325da818b7f097ec7004d453c83b10205167100"
   end
 
+  resource "csound-manual-html" do
```

Comment:
If the doc can be built & installed from the standard archive, we'll do that. If not, let's not add to the maintenance burden of the formula by adding additional resources (especially three versions of the doc).

Reply:
Alas, the Csound’s docs are in a separate repository (https://github.com/csound/manual). I’ve removed all this.